code/modules/admin/verbs/debug.dm

```diff
@@ -357,76 +357,102 @@
 	for(var/areatype in areas_without_camera)
 		to_world("* [areatype]")
 
-/client/proc/cmd_admin_dress()
+/client/proc/cmd_admin_grab_observers()
 	set category = "Fun"
-	set name = "Select equipment"
+	set name = "Grab Observers"
+	set desc = "Grabs players that are observing and spawns them as basic humans beneath your feet."
+
+	var/list/chosen_observers = list()
+	var/next_observer = "NotGeeves"
+	while(next_observer != "== Finished ==")
+		var/list/valid_choices = player_list
+		for(var/choice in valid_choices)
+			if(choice in chosen_observers)
+				valid_choices -= choice
+			if(!istype(choice, /mob/abstract/observer))
+				valid_choices -= choice
+		valid_choices += "== Finished =="
+		next_observer = input("Choose an observer you want to add to the list.", "Choose Observer") as null|anything in valid_choices
```

Comment:
I would personally just take all observers in a certain radius, it would be much cleaner and admins would be able to just go "if you wanna get spawned come to me"

Reply:
ehh idk i think it'd be easier to select from the observer list, since ghost players still want to buzz around and observe stuff until they get put into a body